### DISCUSSION TITLE
Improve DevEx by linking to a page that has details with explanations for common errors

### DISCUSSION THREAD

**Discussion by dylanjha** [muxinc/media-chrome#830] [Proposals]
If a developer uses `media-chrome` incorrectly or runs into a particular kind of error state, we should surface a link to a markdown page that has more details.

I noticed this with nextjs, errors say something like "See here for more info" with a link to a markdown page like this: https://github.com/vercel/next.js/blob/canary/errors/404-get-initial-props.md

The page shows **why this error occurred**, **possible ways to fix it**, and **additional resources**

This is one of those minor DevEx details that really warms my heart, I thought it was neat.

**Comment by heff** [comment]
@dylanjha I want this to be a closable issue. Wanna suggest a first error to do this on and start the trend?

**Comment by cjpillsbury** [comment]
@dylanjha bump on this.
